third_party/amd/include/TritonAMDGPUTransforms/Passes.td

```diff
@@ -65,6 +65,18 @@ def TritonAMDGPUOptimizeEpilogue : Pass<"tritonamdgpu-optimize-epilogue", "mlir:
 
 }
 
+def TritonAMDGPUHoistLayoutConversions : Pass<"tritonamdgpu-hoist-layout-conversions", "mlir::ModuleOp"> {
```

Comment:
Can we define this as a `mlir::FuncOp` pass? You need to change `triton_amd.cc` accordingly. See how it's done for `TritonAMDGPUCanonicalizePointers`.